pipelines/DarkEnergyCamera/RunIsrWithoutInterChipCrosstalk.yaml

```diff
@@ -0,0 +1,9 @@
+description: Run IsrTask for DECam with only intra-chip crosstalk. Inter-chip crosstalk needs pre-prepared crosstalkSources.
+instrument: lsst.obs.decam.DarkEnergyCamera
+tasks:
+  isrOscan:
```

Comment:
Originated with Chris W, but I like it because it distinguishes this "do overscan correction to prepare the crosstalk sources, which technically means running a mini IsrTask" ISR from "real" ISR.

Reply:
I'm confused. Isn't this the old workflow, where you just have "real" ISR?

Reply:
Oh sorry, I got them mixed up, even with the extremely explicit naming. Yeah, I think we can just call it isr.